packages/cryptoassets/src/currencies.ts

```diff
@@ -3021,6 +3021,34 @@ const cryptocurrenciesById: Record<string, CryptoCurrency> = {
       },
     ],
   },
+  ethereum_goerli: {
+    type: "CryptoCurrency",
+    id: "ethereum_goerli",
+    coinType: 1,
```

Comment:
Shouldn't be `60`?

Reply:
My bad, all testnets should be 1 per https://github.com/satoshilabs/slips/blob/master/slip-0044.md :slightly_smiling_face: